run/john.conf

```diff
@@ -391,6 +391,14 @@ Frequency = 141
 Frequency = 135
 #TargetRounds = 1000000
 
+# md5crypt and phpass use same bitstream. Design tools reported
+# possible frequency is 202 MHz. Tested boards run OK at 180 MHz.
+[ZTEX:md5crypt]
+Frequency = 180
+
+[ZTEX:phpass]
+Frequency = 180
+#TargetRounds = 2048
```

Comment:
I just did some testing with wrong `TargetRounds`. The performance impact of running with 8192 against 2048 rounds hashes is about 2%. Running with 2048 against 8192 rounds hashes results in timeouts - so that's what most(?) users will experience, and due to the warning will hopefully adjust the setting. Running with 4096 results in the performance impact for 2048 rounds hashes reduced to 1%, and the timeouts for 8192 rounds hashes are mostly gone - but not entirely gone. Also, I guess this might interfere with `--test` benchmark, where we use 2048 rounds hashes. So I guess let's leave things as-is for now.

Reply:
OK. Performance depends on host system, its CPU and if it's busy with anything at moment of testing. USB communication is CPU-intensive.
What I think is worth consideration: extend no-warning range. Let's say there should be no warning if hashes are 2 times heavier than config's TargetRounds, and if hashes are 4-8 times weaker than config's TargetRounds (since there's not much performance loss anyway).
Currently it warns if hashes are different by more than 2X from TargetRounds in either direction.